ci-operator/jobs/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.4-presubmits.yaml

```diff
@@ -0,0 +1,149 @@
+presubmits:
+  openshift/assisted-image-service:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-ocm-2\.4$
+    - ^release-ocm-2\.4-
```

Comment:
For hotfixes of some sort I guess?
I suppose we won't have `2.4.z` branches?

Reply:
We currently do z releases off the `ocm-2.3` branch so I thought it would be the same for `release-ocm-2.4`